Procfile

```diff
@@ -1,2 +1,3 @@
 web: bundle exec unicorn -p $PORT -c ./config/unicorn.rb
-worker:  bundle exec rake jobs:work
\ No newline at end of file
+worker:  bundle exec rake jobs:work
+mailcatcher: mailcatcher -f
```

Comment:
Wouldn't this attempt to run `mailcatcher` in Heroku? I'd only put this locally in a `Procfile.dev` that can be run with `foreman -f Procfile.dev`